config/v1/types_infrastructure.go

```diff
@@ -170,6 +174,28 @@ type BareMetalPlatformStatus struct {
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
 }
 
+// OpenStackPlatformStatus holds the current status of the OpenStack infrastructure provider.
+type OpenStackPlatformStatus struct {
+	// apiServerInternalIP is the IP address behind the
+	// InfrastructureStatus' apiServerInternalURL field. OpenStack and
+	// other platforms without a readily available DNS-as-a-service
+	// use it before the DNS for the nodes themselves is set up.
+	APIServerInternalIP string `json:"apiServerInternalIP,omitempty"`
```

Comment:
does this field parallel, the one for baremetal?  if so, make the comments consistent.